entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+source=${SOURCE:-.}
+
+cd "${GITHUB_WORKSPACE}"/"${source}" || exit
+
+tcctl message --text "Start tagger"
+
+NEW_GIT_TAG=$(tcctl helpers generate_increment_version)
+
+tcctl message --text "The new tag is: $NEW_GIT_TAG"
+
+remote=$(git config --get remote.origin.url)
+repo=$(basename "$remote" .git)
+commit=$(git rev-parse HEAD)
```

Comment:
You are created single command for this.

Reply:
My command is not suitable for pushing from a container to git actions - here authorization by token key...
If only I improve it ...